gradle-enterprise-conventions-plugin/gradlew

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```

Comment:
:o: I wonder if we should just use the wrapper from the main project here to avoid multiple places that require updates. 

This can be done by replacing this wrapper script here with:

```
  
#!/usr/bin/env sh

exec "$(dirname "$0")/../gradlew" "$@"
```

Reply:
This is moved to a separate repository: https://github.com/gradle/gradle-enterprise-conventions-plugin